joomla/joomla-cms
# Replaced inline JavaScript in wrapper

#3849 rewritten

---
### How to test
- add a new wrapper
- see that the iframe looks good in frontend
- appy this patch
- make sure it works still expected :smile:
